oak_functions/loader/src/tests.rs

```diff
@@ -68,23 +72,29 @@ fn bench_wasm_handler(bencher: &mut Bencher) {
             fs::read(TEST_WASM_MODULE_PATH).expect("Couldn't read test Wasm module");
         let wasm_handler =
             WasmHandler::create(&wasm_module_bytes).expect("Couldn't create the server");
-        let request = hyper::Request::builder()
-            .method(http::Method::GET)
-            // We don't really send the request. So the URI can be anything. It is only needed for
-            // building a valid request.
-            .uri("example.com")
-            .body(hyper::Body::empty())
-            .unwrap();
-        bencher.iter(|| wasm_handler.handle_request(&request));
+        bencher.iter(|| {
+            let request = hyper::Request::builder()
+                .method(http::Method::POST)
+                // We don't really send the request. So the hostname can be anything. It is only
+                // needed for building a valid request.
+                .uri("http://example.com/invoke")
+                .body(hyper::Body::empty())
+                .unwrap();
+            let resp = tokio::runtime::Runtime::new()
```

Comment:
I think the purpose of this benchmark was to just measure the loading of the wasm module. This is now processing the request and producing the response too. Is this the intended behaviour?

Reply:
We want it to measure something meaningful from the point of view of a user of the Oak Functions Loader, so I think the response processing time is what matters (though we may want more precise smaller benches too for individual things like loading Wasm module at some point? not sure that's super useful at this stage by itself).

Also I don't think the purpose of the bench was to measure the loading of the wasm module, since it was already measuring the `handle_request` method, right? Or, if that was the intention, perhaps the naming was not quite accurate?

Reply:
In fact, I don't think it was measuring only the loading of the module, it was definitely also measuring the invocation of the Wasm exported function (since they are both in `handle_request`). I don't think it was even possible to measure Wasm module loading without the actual invocation, or did I misunderstand your comment?

Reply:
The actual invocation (via `invoke_export`) does not do much in the case of this minimal Wasm module, and the response was hard-coded. So, I think the time that was measured previously was mostly just loading the module (even though this was done via the call to `handle_request`). I agree that if the intention is to measure loading of the module, some refactoring and renaming would be needed. 
But I am fine with this test if actually benchmarking the response processing time is intended.

Reply:
What I don't understand is why does it take so much longer now? There should be the same amount of processing going on, right? That's why I suspect some part of the execution was just being optimized out earlier for some reason. Or maybe creating the tokio executor also takes longer?

I am also still not 100% sure I understand your point (sorry, just want to make sure I am not overlooking something):

> So, I think the time that was measured previously was mostly just loading the module (even though this was done via the call to handle_request)

Isn't this the same now? AFAICT the same exact Wasm module is loaded **and** invoked, both before and after this PR.

Reply:
Oops, I just realised I was super wrong!

The overhead is almost entirely due to creating the tokio runtime inside the benchmark iteration! I now moved it out and the bench time went down to 25ms! All my assumptions and conclusions were incorrect :)

Reply:
Nice. I did not know creating the runtime takes that much time.